pkgs/tools/security/tcb/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl
+, linux-pam, libxcrypt
+}:
```

Comment:
```suggestion
{ lib
, stdenv
, fetchurl
, linux-pam
, libxcrypt
}:

Reply:
Can I keep the current formatting? :)
The other suggested variants have been fixed.

Reply:
One of the reasons we use one attribute per line is better looking diffs.

When in the future a dependency is added or removed, it generates one line for diff, instead of two.